src/editpage/components/response/MessagesWindow.vue

```diff
@@ -180,6 +180,14 @@ export default {
           error: this.addError.message,
           file: this.addError.file,
         });
+        this.showTimeout = setTimeout(() => {
```

Comment:
Do you need this `setTimeout`?
Also it will override `this.showTimeout` of the error method above which could lead to missing erros.

Reply:
So do ```this.showTimeout(() =>``` instead?

Reply:
No I think nothing without timeout just the `requestNextAnimationFrame(() => ...`

But do test it, I'm just guessing it's not needed

Reply:
It works without the ```setTimeout`` and is a lot sharper and without delays, I'll change that too

Reply:
Nice!